index.html

```diff
@@ -5,21 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
     <title>Portfolio Website</title>
 </head>
 <body >
     <nav class="header-logo">
         <a href="#"><img src="images/My Logo.svg" alt="" class="logo"></a>
-        <img src="images/Enabled.png" alt="" class="image-icon"></nav>
+        <img src="images/Enabled.png" alt="" class="image-icon">
+         <div class="name-1">
+            <!-- <div id="close_button">x</div> -->
+            <img src="images/Closing-cross.png" alt="" id="close_button">
+            <!-- <div><i class="fa-solid fa-xmark"></i></div> -->
+            <ul class="mobile-Menu-list">
+              <li><a href="#works-container">Portfolio</a></li>
+              <li><a href="#about-myself">About</a></li>
+              <li><a href="#contact-form">Contact</a></li>
+            </ul>
```

Comment:
- [ ] Your design is almost exceptional @MarkYegon7, But Please notice on the bottom of your menu there is a horizontal line that can be used to close the menu `you are not expected to implement the functionality yet` but the design should be the same as the figma design which means we need to add the line.

### Figma design.
![image](https://user-images.githubusercontent.com/69077061/225386695-b6d14ce3-d913-4bda-a4d7-e1b8e0a519af.png)
- [ ] I notice on large screen devices your menu does not take the full height of the page. Please make sure the menu always takes `100%` of the mobile height.
### Your menu on `Samsung Galaxy 20 ultra`
![image](https://user-images.githubusercontent.com/69077061/225387638-e83228fc-7c09-4514-ac80-4e2d742dd311.png)